src/client/Microsoft.Identity.Client/TelemetryCore/Http/HttpTelemetryManager.cs

```diff
@@ -125,17 +125,13 @@ public string GetCurrentRequestHeader(ApiEvent eventInProgress)
 
             // Since regional fields will only be logged in case it is opted.
             var regionalFields = new StringBuilder();
-            if (!string.IsNullOrEmpty(regionDiscovered))
-            {
-                regionalFields.Append(regionDiscovered);
-                regionalFields.Append(",");
-                regionalFields.Append((regionSource));
-                regionalFields.Append(",");
-            }
+            regionalFields.Append(regionDiscovered);
```

Comment:
Looks good, but can you please fix the `GetLastRequestHeader` as well?

Reply:
Fixed that too.